src/producer.py

```diff
@@ -0,0 +1,38 @@
+from kafka import KafkaProducer
+import logging
+import uuid
+
+from constants import PRODUCER_API_VERSION
+
+class Producer(object):
+    """Wrapper for a Kafka Producer
+    :param host: Hostname of the bootstrap server
```

Comment:
not entirely sure what methodology of a doc string this approach follows, but for readability's sake what do you think about changing this to something like:

```python

"""
Wrapper for a Kafka producer
    ----
    host (str): Hostname of the bootstrap server
    port (str): Port of bootstrap server
"""
```

Just an idea. I might very well just be unaware of this type of doc string annotation.

Reply:
https://sphinx-rtd-tutorial.readthedocs.io/en/latest/docstrings.html